daemon/web/components/buttons/ShutdownButton/index.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+
 
 import '../index.sass';
 
-const ShutdownButton = ({ style }) => (
+const ShutdownButton = () => (
   <div>
-    <button className="button" type="button" style={style}>
+    <button className="button" type="button">
 Shut Down Containers
     </button>
   </div>
 );
 
 ShutdownButton.propTypes = {
```

Comment:
since there are no props, this can be removed entirely :)

Reply:
WAIT i think you changed the wrong button? this is `ShutdownButton`

Reply:
should probably be updating `LogoutButton` instead

Reply:
Okay lol yep.. Should it be removed for all the buttons? all the buttons now don't follow the global style classes

Reply:
I think we should leave it for the other buttons for now, until we can check if they use it or not.

The buttons as a whole probably need a rework - we probably don't want `ShutdownButton` and `LogoutButton`, but a `PrimaryButton` and `SecondaryButton` instead, to improve re-usability - but this is fine for the time being.

Reply:
So yeah, just remove it for `LogoutButton`, and leave it as is for the others 🚀

Reply:
OKayy 👍